ZIP 239: Clarify the behaviour of zcashd and the intended behaviour for unrecognized inventory types (#545)
zcash/zips: zip-0239.rst

network upgrade.

Nevertheless, it is possible for a node to receive an ``inv`` or ``getdata`` message

> containing an inventory entry of type ``MSG_WTX``, on a peer connection that has
> negotiated protocol version 170014 or later, before NU5 has activated. The node MUST
> handle this case in the same way that it would after NU5 activation when it has no
> v5 transactions to relay.

Reviewer: ```suggestion
containing an inventory entry of type ``MSG_WTX``, on a peer connection that has
negotiated protocol version 170014 or later, before NU5 has activated. In this case, the node MUST NOT advertise, fetch, or provide v5 transactions.
```

.. [#bip-0339] `BIP 339: WTXID-based transaction relay <https://github.com/bitcoin/bips/blob/master/bip-0339.mediawiki>`_
.. [#p2p-inv] `Bitcoin Developer Reference: P2P Network — Inv <https://developer.bitcoin.org/reference/p2p_networking.html#inv>`_
.. [#p2p-getdata] `Bitcoin Developer Reference: P2P Network — GetData <https://developer.bitcoin.org/reference/p2p_networking.html#getdata>`_

> .. [#zcashd-propagation] `zcashd book: P2P data propagation <https://zcash.github.io/zcash/design/p2p-data-propagation.html>`_

Author: This is not yet a valid URL, but it will be soon.
